pkg/apis/feeds/v1alpha1/feed_types.go

```diff
@@ -40,6 +42,11 @@ type Feed struct {
 	Status FeedStatus `json:"status"`
 }
 
+var _ apis.Validatable = (*Feed)(nil)
+var _ apis.Defaultable = (*Feed)(nil)
+var _ apis.Immutable = (*Feed)(nil)
+var _ webhook.GenericCRD = (*Feed)(nil)
```

Comment:
No `runtime.Object` cast here?

Reply:
so.. some egg on my face... webhook.GenericCRD  has all of these

Reply:
I could boil it down to just webhook.GenericCRD